license-compliance/gradle-license-report/license-normalizer-bundle.json

```diff
@@ -582,6 +567,10 @@
     {
       "bundleName": "mit-or-mpl-2.0",
       "licenseNamePattern": "^MIT License; Mozilla Public License 2\\.0 \\(MPL 2\\.0\\)$"
+    },
+    {
+      "bundleName": "cddl-1.1-or-gpl-2",
```

Comment:
question: Have you checked this normalization locally? In the report you shared there was no license URL. I am not sure if this pattern will work in this case

Reply:
Yes, I've checked: plugin recognizes this license. This URL is located in the MANIFEST.MF file.